.github/workflows/vrt.yml

```
@@ -4,6 +4,9 @@ on:
     push:
       branches:
         - main
+        - !changeset-release/**
+        - !dependabot/**
```

```suggestion
        - "!changeset-release/**"
        - "!dependabot/**"
```
I think these might have to be quoted! Otherwise, looks great!

---

I think you're right. I see a linter error warning showing up in my file now.